_posts/2022-11-10-why-python-package-health-metrics-matter.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: single
+title: "Why Thinking About Metrics Should be the First Thing That Any New Organization, Program or Project Does"
+excerpt: "Collecting baseline data that aligns with the goals and outcomes of your project, program or organization is critical to do at the beginning. Here I briefly explain why Python package health metrics are so important to the long-term success of pyOpenSci."
+author: "Leah Wasser"
+permalink: /blog/why-python-package-health-metrics-matter
+header:
+    overlay_color: "#666"
+    overlay_filter: 0.6
+categories:
+  - blog-post
+  - highlight
+  - python-packaging
+  - peer-review
+toc: true
+comments: true
+---
+
+
+## Metrics are critical to the development of any program 
+
+I've created a few open science focused programs now from the ground up. One at 
+NEON and another at CU Boulder. When building a new program, one of the first 
+things that I do (after defining the mission and goals) is to define the metrics 
+that constitute success. 
+
+These metrics are critical to define early because:
+
+* They drive everything that you do
```

Comment:
```suggestion
* They drive everything that you do.
```
Suggest either making all of these bullet points sentences with periods or none of them, but not mixing